instrumentation/jmx-metrics/library/src/test/java/io/opentelemetry/instrumentation/jmx/engine/RuleParserTest.java

```diff
@@ -616,7 +641,7 @@ void testNoBeans() {
             + "  - mapping:           # still no beans\n"
             + "      A:\n"
             + "        metric: METRIC_NAME\n";
-    runNegativeTest(yaml);
+    runNegativeTest(yaml, "No ObjectName specified");
```

Comment:
[for reviewer] previously any parsing error would make the test pass, adding the expected exception message allows to ensure that we trigger the validation error we expect.